android/app/src/main/res/values-es/strings.xml

```diff
@@ -587,8 +587,10 @@
 	<string name="learn_more">Leer más</string>
 	<!-- Subway exits for public transport marks on the map -->
 	<string name="core_exit">Salida</string>
-	<string name="routing_add_start_point">Buscar un origen para el plan de ruta</string>
-	<string name="routing_add_finish_point">Buscar un destino para el plan de ruta</string>
+	<!-- User selected the destination by pressing Route To, but the current position is unknown. User needs to select a starting point of a route using search or by tapping on the map and then pressing "Route From". -->
+	<string name="routing_add_start_point">Utiliza la búsqueda o toca en el mapa para añadir un punto de inicio de ruta</string>
+	<!-- User selected the start of a route by pressing Route From. Now the destination of a route should be selected using search or by tapping on the map and then pressing "Route To". -->
+	<string name="routing_add_finish_point">Utiliza la búsqueda o toca en el mapa para añadir un punto de destino</string>
```

Comment:
I would keep it simple: "Utiliza la búsqueda o toca en el mapa para añadir el destino",
 or "Busca o toca en el mapa para añadir el destino"